README.md

```diff
@@ -1,5 +1,7 @@
 ![headline](docs/assets/headline.png)
 
+> 🏖 From W/C 11/7 due to Summer Vacations of some of the maintainers, expect the project to move a little slower than normal, and support to be limited. Normal service will resume on W/C 8/8! 🏝
```

Comment:
Honestly I can't parse "W/C 11/7" for several reasons :) Might want to type that out including month name, in both places.

Reply:
I must confess I had to google that ✋